app/src/main/java/com/guruthedev/instagram/utils/Constants.kt

```diff
@@ -0,0 +1,10 @@
+package com.guruthedev.instagram.utils
+
+ object Constants{
+    const val PREF_NAME = "Login_Preference"
+    const val IS_LOGIN = "is_Sign"
```

Comment:
can we maintain a common style 
full_name ,is_Sign has underscore 
username, email is all smallCaps